daemon/src/maker_cfd.rs

```diff
@@ -217,6 +217,13 @@ where
 
         self.tasks.add(rollover_actor_future);
 
+        self.takers
+            .send(RollOverProposed {
+                order_id: proposal.order_id,
+                address: rollover_actor_addr.clone(),
+            })
+            .await?;
```

Comment:
@da-kami: I had to reintroduce this because we still need to tell the `maker_inc_connections::Actor` where to forward messages from the taker about the current rollover of this particular CFD. This sucks, but it may be solved down the line with QUIC's multiplexing which Thomas has been talking about.

Reply:
hm... would it be possible to do this from the rollover actor instead? 
If not easily possible we can leave it here. Would it be worth adding a ticket describing that problem then?

Reply:
I think moving it to the rollover actor makes more sense, let me try.